docs/releasenotes/7.0.0.rst

```diff
@@ -85,7 +85,7 @@ Custom unidentified image error
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Pillow will now throw a custom ``UnidentifiedImageError`` when an image cannot be
-identified. For backwards compatibility, this will inherit from ``IOError``.
+identified. For backwards compatibility, this will inherit from ``OSError``.
```

Comment:
@radarhere What do you think about this one?

Reply:
I would think that's fine to change, because Pillow 7 dropped support for Python 2.7, so both errors are synonymous at this point.